ibm/mas_devops/roles/suite_app_install/README.md

```diff
@@ -107,11 +107,10 @@ Optional. Defines what plan will be used in application install.
 Role Variables - Visual Inspection Configuration
 -------------------------------------------------------------------------------
 ### mas_app_settings_visualinspection_storage_class
-Storage class used for user data. This must support ReadWriteOnce
+Optional - Storage class used for user data. This must support ReadWriteOnce
 
-- Optional
 - Environment Variable: `MAS_APP_SETTINGS_VISUALINSPECTION_STORAGE_CLASS`
-- Default: Defaults to `ibmc-file-gold` if the storage class is available in the cluster.
+- Default: Auto-selected from storage classes installed in the cluster.
```

Comment:
MVI EFS requires uid:`999` due to mongoldb requirement and Cluster EFS requires uid:`0';
Auto selection would 100% fail MVI would fail with `Could not bind PVC`

```kind: StorageClass
apiVersion: storage.k8s.io/v1
metadata:
  name: mvi-efs
  uid: 85441182-8403-4793-a34c-04ae0b0e1350
 provisioner: efs.csi.aws.com
parameters:
  basePath: /mvi
  directoryPerms: '777'
  fileSystemId: fs-00c7e21665962211a
  gid: '0'
  provisioningMode: efs-ap
  uid: '999'
reclaimPolicy: Delete
volumeBindingMode: Immediate```

Cluster EFS requires aid:'0'
```kind: StorageClass
apiVersion: storage.k8s.io/v1
metadata:
  name: efs
  uid: 85441182-8403-4793-a34c-04ae0b0e1350
 provisioner: efs.csi.aws.com
parameters:
  basePath: /rosa
  directoryPerms: '777'
  fileSystemId: fs-00c7e21665962211a
  gid: '0'
  provisioningMode: efs-ap
  uid: '0'
reclaimPolicy: Delete
volumeBindingMode: Immediate```